# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
joint_limits:
  ra_elbow_joint:
    has_velocity_limits: true
    max_velocity: 3.15
    has_acceleration_limits: false
    max_acceleration: 0
    min_position: -2.5
    max_position: 0.0
  ra_shoulder_lift_joint:
    has_velocity_limits: true
    max_velocity: 2.16
    has_acceleration_limits: false
    max_acceleration: 0
    min_position: -3 #-2.39
    max_position: -0.3
  ra_shoulder_pan_joint:
    has_velocity_limits: true
    max_velocity: 2.16
    has_acceleration_limits: false
    max_acceleration: 0
    min_position: 0.4
    max_position: 3.5 #2.75
  ra_wrist_1_joint:
    has_velocity_limits: true
    max_velocity: 3.2
    has_acceleration_limits: false
    max_acceleration: 0
    min_position: -2.74
    max_position: 0.47 #0.35
  ra_wrist_2_joint:
    has_velocity_limits: true
    max_velocity: 3.2
    has_acceleration_limits: false
    max_acceleration: 0
    min_position: -1.88
    max_position: 2.5
  ra_wrist_3_joint:
    has_velocity_limits: true
    max_velocity: 3.2
    has_acceleration_limits: false
    max_acceleration: 0
    min_position: -6.2
    max_position: 0.72
  rh_FFJ1:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_FFJ2:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_FFJ3:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_FFJ4:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_LFJ1:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_LFJ2:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_LFJ3:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_LFJ4:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_LFJ5:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_MFJ1:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_MFJ2:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_MFJ3:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_MFJ4:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_RFJ1:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_RFJ2:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_RFJ3:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_RFJ4:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_THJ1:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_THJ2:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_THJ3:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_THJ4:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
  rh_THJ5:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: false
    max_acceleration: 0
